src/nuget/Microsoft.DotNet.Build.Tasks.Feed.nuspec

```diff
@@ -16,7 +16,9 @@
     <copyright>Copyright © Microsoft Corporation</copyright>
     <tags></tags>
     <dependencies>
-     <!-- Do not add any dependencies to this package as we want to install it versionless (i.e. -ExcludeVersion) and we don't want the dependencies to be installed that way -->
+     <group targetFramework=".NETStandard1.5">
```

Comment:
I'm good either way. Either a package dependency or including the dll, I will leave it to you.

Reply:
I'll go with the package ref